qualtran/bloqs/phase_estimation/lp_resource_state_test.py

```diff
@@ -38,11 +38,6 @@ def test_lp_resource_state_auto(bloq_autotester):
     bloq_autotester(_lp_resource_state_small)
 
 
-def test_lp_resource_state_symb():
-    bloq = _lp_resource_state_symbolic.make()
-    assert bloq.t_complexity().t == 4 * bloq.bitsize
-
-
```

Comment:
perhaps we should keep this test marked `@pytest.mark.xfail`? (so that we don't forget to fix symbolics for LPRS.)

Reply:
SGTM, Done.

Reply:
It is not ideal to have the symbolis not working -- maybe it's best to wait till we can add a CRz  Bloq and use that so we can continue to support symbolics. 

I don't think there's any rush to remove t_complexity; especially by breaking symbolic resource counting which is a lot more useful.

Reply:
Reverted this and reenabled symbolics.